test/highlevel.cxx

```diff
@@ -12,6 +12,25 @@
  */
 namespace fire::test {
 
+/**
+ * need test class to make sure schema evolution
+ * interacts well with copy
+ */
```

Comment:
```suggestion
 * WARNING: We only test that the copy functions for copying while keeping the same class version.
 * This makes sense because how would we evolve the schema without knowing what the new schema is?
 */
```